modules/check_in/spec/sidekiq/travel_claim_base_job_spec.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe CheckIn::TravelClaimBaseJob do
+  let(:mobile_phone) { '202-555-0123' }
+  let(:appointment_date) { '2023-05-15' }
+  let(:claim_number) { '1234' }
+  let(:facility_type) { 'oh' }
+  let(:template_id) { 'template-id-123' }
+  let(:job_opts) do
+    {
+      mobile_phone:,
+      appointment_date:,
+      template_id:,
+      claim_number:,
+      facility_type:
+    }
+  end
+
+  describe '#send_notification' do
+    subject { described_class.new.send_notification(job_opts) }
+
+    before do
+      allow_any_instance_of(VaNotify::Service).to receive(:send_sms)
+      allow_any_instance_of(described_class).to receive(:logger).and_return(Logger.new(nil))
+    end
+
+    it 'logs the notification and sends SMS' do
```

Comment:
It looks to me this test is not needed as the happy day scenario is tested starting in line 62. I also don't like the expects on the class under test as well. Perhaps delete this.